### PR TITLE
CCTRI 1318/tweak documentation

### PR DESCRIPTION
**Pull request by GeorgiyII**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,289 @@
...
+  single response per each requested observable.
+  - Applies to the following CTIM entities:
+    - `Judgement`,
+    - `Sighting`,
+    - `Indicator`.
+  - Must be a positive integer. Defaults to `100` (if unset or incorrect).
+
+## CTIM Mapping Specifics
+
+To create CTIM entities using data from `search` and `result` endpoints.
```
Comment (gdavoian) [inline README.md:280]: Didn't get the idea of the phrase.

File: README.md [hunk 2]
```diff
@@ -1 +1,289 @@
-tr-05-serverless-urlscanio
+[![Travis CI Build Status](https://travis-ci.com/CiscoSecurity/tr-05-serverless-urlscanio.svg?branch=develop)](https://travis-ci.com/CiscoSecurity/tr-05-serverless-urlscanio)
+
```
Comment (gdavoian) [inline README.md:2]: 1 line-break is enough here :)

Comment (GeorgiyII) [reply README.md:2]: done

File: README.md (hunk 1 above)
Comment (GeorgiyII) [reply README.md]: rephrased this

File: README.md [hunk 3]
```diff
@@ -1 +1,288 @@
...
+- `ip`
+- `ipv6`
+- `domain`
+- `url`
+
+### JWT Payload Structure
+
+```json
+{
+  "key": "<urlscan.io_API_KEY>"
```
Comment (gdavoian) [inline README.md:262]: The placeholder should be in the upper case, i.e. "<URLSCAN.IO_API_KEY>" :)

I'm approving the PR, the grammar will be checked by @ivlana.

Comment (GeorgiyII) [reply README.md]: done

**Pull request merged**